cluster-stamp.json

```diff
@@ -479,7 +482,7 @@
                         "name": "apw-frontend-ip-configuration",
                         "properties": {
                             "PublicIPAddress": {
-                                "id": "[resourceId(subscription().subscriptionId, variables('vNetResourceGroup'), 'Microsoft.Network/publicIpAddresses', 'pip-BU0001A0008-00')]"
+                                "id": "[resourceId(subscription().subscriptionId, variables('vNetResourceGroup'), 'Microsoft.Network/publicIpAddresses', '[concat('pip-BU0001A0008-00', parameters('teamName')]')]"
```

Comment:
should we add a hyphen (-) between the "bu" id and the team name?  Also, do we need the "bu" name or is it hard to remove?